dev-util/pycharm-community/pycharm_community_bin-2023.2.3.recipe

```diff
@@ -1,14 +1,17 @@
 SUMMARY="A Lightweight IDE for Python & Scientific development"
 DESCRIPTION="PyCharm is a Python IDE for professional developers by JetBrains"
 HOMEPAGE="https://www.jetbrains.com/pycharm/"
-COPYRIGHT="2020-2021 JetBrains s.r.o."
+COPYRIGHT="2020-2023 JetBrains s.r.o."
 LICENSE="Apache v2"
 REVISION="1"
 SOURCE_URI="https://download.jetbrains.com/python/pycharm-community-$portVersion.tar.gz#noarchive"
-CHECKSUM_SHA256="10331302819324794e70843a73bfedbc6023778e4374f57f80608983be4e7a11"
+CHECKSUM_SHA256="d59dd88c1eb51cdd756433d415588c573ca944ebf6f08844b8ac8cd2e3d9937b"
 ADDITIONAL_FILES="PyCharm_Logo.hvif"
 
 ARCHITECTURES="any"
+if [ "$targetArchitecture" = x86_gcc2 ]; then
+	ARCHITECTURES="?any"
+fi
```

Comment:
This doesn't prevent from being able to check a package from the x86_64 buildmaster, but holds off on creating one for 32bit.

Reply:
pycharm is unsupported upstream on 32-bit since 2019.

Reply:
Then I guess it's better to use "?all x86_64" for now here (and for NetBeans). 
Come to think of it, I could launch PyCharm without problems, but I couldn't get it to connect to the interpreter, so let's wait a while on this one.